src/boot/aqrbootsetup.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# project aquarium's boot setup
+# Copyright (C) 2021 SUSE, LLC.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+
+function overlay() {
+  lower=${1}
+  upper=${2}
+
+  aqrdir=/aquarium/${upper}
+  mount -t overlay \
+    -o lowerdir=${lower},upperdir=${aqrdir}/overlay,workdir=${aqrdir}/temp \
+    overlay ${lower} || return 1
+}
+
+systemdisk=$(lvm lvs --noheadings -o vg_name,lv_name @aquarium |
+             sed 's/^[ ]\+//' | tr ' ' '-')
+
+[[ -z "${systemdisk}" ]] && \
+  echo "Aquarium system disk not found, assuming new deployment" && \
+  exit 0
+
+[[ "${systemdisk}" != "aquarium-systemdisk" ]] && \
+  echo "Unknown system disk, expected aquarium-systemdisk" >/dev/stderr && \
+  exit 1
+
+[[ ! -d "/aquarium" ]] && \
+  (mkdir /aquarium || (echo "error creating /aquarium" && exit 1))
+
+[[ ! -e "/dev/mapper/aquarium-systemdisk" ]] && \
+  echo "Unable to find aquarium block device" && \
+  exit 1
+
+# mount system disk
+mount /dev/mapper/aquarium-systemdisk /aquarium
```

Comment:
```suggestion
mount /dev/mapper/aquarium-systemdisk /var/lib/aquarium
```

LSB?

Reply:
ugh, this would have been a good idea, but will likely need to be done in a next pass -- we'll need to change this on the creation code as well. I'll open an issue and address it when tackling the cluster extension bits.